nucypher/characters/control/specifications/fields/datetime.py

```diff
@@ -17,8 +17,10 @@
 
 import maya
 from marshmallow import fields
+from pendulum.parsing import ParserError
```

Comment:
May be better to just use `maya.pendulum.parsing.ParserError`. When one package raises an exception from another package, it looks weird (`maya` could have wrapped it, but since they decided not to...).

Reply:
✔️